default.json

```diff
@@ -10,7 +10,8 @@
       {
          "groupName": "all dependencies",
          "matchPackageNames": [ ".*" ],
-         "minimumReleaseAge": "7 days"
+         "minimumReleaseAge": "7 days",
+         "internalChecksFilter": "strict"
```

Comment:
Just checking, isn't [`strict` the default for `internalChecksFilter`](https://docs.renovatebot.com/configuration-options/#internalchecksfilter)? Additionally, it doesn't seem `internalChecksFilter` is supported on `packageRules` items ([docs](https://docs.renovatebot.com/configuration-options/#packagerules), [types](https://github.com/renovatebot/renovate/blob/9f600d14457b3d99b08b255420c56cfda7dd2bf2/lib/config/types.ts#L362)).

Reply:
@onebytegone 
Regarding if the field is supported or not, my TS is a bit rusty, but I believe it allows any key-value pair because of the `extends... Record<string, unknown>` in [line 365](https://github.com/renovatebot/renovate/blob/9f600d14457b3d99b08b255420c56cfda7dd2bf2/lib/config/types.ts#L365). The same happens in the [RenovateConfig interface](https://github.com/renovatebot/renovate/blob/9f600d14457b3d99b08b255420c56cfda7dd2bf2/lib/config/types.ts#L224C8-L224C18), The usage of `internalChecksFilter` is covered by the [package tests](https://github.com/renovatebot/renovate/blob/9f600d14457b3d99b08b255420c56cfda7dd2bf2/lib/workers/repository/process/lookup/filter-checks.spec.ts#L182-L183).
But I've now searched through their tests, and it seems it should also work without defining it. However, I couldn't find where the default value is defined.

I've search some other options and there seems to be a problem with our regex for the matchPackageNames. It seems more likely to be the cause of the problem, so I'll stash these changes locally and commit a new one with the regex fix